worker/src/main/java/gov/cms/ab2d/worker/processor/coverage/CoverageMappingCallable.java

```diff
@@ -53,11 +63,12 @@
     private int pastReferenceYear;
     private final Map<Integer, Integer> referenceYears = new HashMap<>();
 
-    public CoverageMappingCallable(FhirVersion version, CoverageMapping coverageMapping, BFDClient bfdClient) {
+    public CoverageMappingCallable(FhirVersion version, CoverageMapping coverageMapping, BFDClient bfdClient, ContractRepository contractRepository) {
```

Comment:
Why not pass in the DTO as part of the CoverageMapping object?

Reply:
Because this will cause us to be dependent on the contract table in the database for contractmapping/coverageperiod to be dependent on the contract table for the data. Defeating the whole purpose of splitting coverage and contract.